.github/workflows/sanitisers.yml

```diff
@@ -99,7 +99,7 @@ jobs:
             ${{ runner.os }}-
       # --- Build with thread sanitising
       - name: "Build tests with address sanitising"
-        run: inv dev.sanitise Address --noclean
+        run: inv dev.sanitise Address faabric_tests --noclean
```

Comment:
This was so that I could sanitise and run other targets such as the distributed tests.